setup.cfg

```diff
@@ -1,16 +1,16 @@
 [flake8]
 max-line-length = 120
-exclude = docs/src, build, dist
+exclude = docs/src, build, dist, .ipynb_checkpoints
 
 [isort]
 line_length = 120
 not_skip = __init__.py
+skip_glob = .ipynb_checkpoints
 known_first_party = pyro, tests
 known_third_party = opt_einsum, six, torch, torchvision
 
 [tool:pytest]
 filterwarnings = error
-    ignore::DeprecationWarning
```

Comment:
@neerajprad I think this line is redundant. What do you think?

Reply:
Can someone explain to me why this is redundant?

Reply:
I believe we should instead remove the `once::DeprecationWarning` line. We really do want to ignore `DeprecationWarning`s

Reply:
Yeah, let me see what's different when we use `ignore` instead of `once`.